### PR TITLE
Fix #2493 by removing the unused joint to camera_rgb_frame

### LINKED ISSUE #2493: Launching nav2_bringup tb3_simulation_launch.py - Gazebo can't find Child Link[camera_rgb_frame]
## Bug report

**Required Info:**

- Operating System:
Ubuntu 20.04
- ROS2 Version:
Foxy binaries
- Version or commit hash:
ros-foxy-navigation2	0.4.7-1focal.20210701.023143
- DDS implementation:
Fast-RTPS

#### Steps to reproduce issue
https://navigation.ros.org/getting_started/index.html#running-the-example
```
source /opt/ros/<ros2-distro>/setup.bash
export TURTLEBOT3_MODEL=waffle
export GAZEBO_MODEL_PATH=$GAZEBO_MODEL_PATH:/opt/ros/<ros2-distro>/share/turtlebot3_gazebo/models
// Add --verbose to the gzserver in the launch file.
// Either change it directly or copy it and run it afterwards with
// ros2 launch tb3_simulation_launch.py
ros2 launch nav2_bringup tb3_simulation_launch.py
```
Result:
```
[gzserver-1] [Msg] Waiting for master.
[gzserver-1] [Msg] Connected to gazebo master @ http://127.0.0.1:11345
[gzserver-1] [Msg] Publicized address: 192.168.122.20
[gzserver-1] [Msg] Loading world file [/opt/ros/foxy/share/nav2_bringup/worlds/waffle.model]
[gzserver-1] [Err] [Joint.cc:297] EXCEPTION: Couldn't Find Child Link[camera_rgb_frame]
[gzserver-1] 
[gzserver-1] [Err] [Model.cc:272] LoadJoint Failed
```

#### Expected behavior
Running without errors

#### Actual behavior
Gazebo can't load the broken `waffle.model` from `nav2_bringup`.

##### Temporary solution
Use:
```
ros2 launch tb3_simulation_launch.py world:=/opt/ros/foxy/share/turtlebot3_gazebo/worlds/turtlebot3_worlds/waffle.model
```

##### Real fix
Change this back (in `tb3_simulation_launch.py`):
From:
```python
declare_world_cmd = DeclareLaunchArgument(
        'world',
        # TODO(orduno) Switch back once ROS argument passing has been fixed upstream
        #              https://github.com/ROBOTIS-GIT/turtlebot3_simulations/issues/91
        # default_value=os.path.join(get_package_share_directory('turtlebot3_gazebo'),
        #                            'worlds/turtlebot3_worlds/waffle.model'),
        default_value=os.path.join(bringup_dir, 'worlds', 'waffle.model'),
        description='Full path to world model file to load')
```
To:
```python
declare_world_cmd = DeclareLaunchArgument(
        'world',
        # ??? TODO(orduno) Switch back once ROS argument passing has been fixed upstream
        # ???             https://github.com/ROBOTIS-GIT/turtlebot3_simulations/issues/91
        default_value=os.path.join(get_package_share_directory('turtlebot3_gazebo'),
                                   'worlds/turtlebot3_worlds/waffle.model'),
        description='Full path to world model file to load')
```

### PR DESCRIPTION
**Pull request by Darkproduct**
It looks like `camera_rgb_link` is now `camera_link` e3ee2da2d5a274b52d32fcdf7e270567ca1093d0

## Basic Info

| Info | Please fill out this column |
| ------ | ----------- |
| Ticket(s) this addresses   | #2493 |
| Primary OS tested on | Ubuntu |
| Robotic platform tested on | none, Gazebo 11.5 |

## Description of contribution in a few bullet points
Fix the `waffle.model` file

## Description of documentation updates required from your changes
None

## Future work that may be required in bullet points
None

#### For Maintainers: 
- [ ] Check that any new parameters added are updated in navigation.ros.org
- [ ] Check that any significant change is added to the migration guide
- [ ] Check that any new functions have Doxygen added
- [ ] Check that any new features have test coverage
- [ ] Check that any new plugins is added to the plugins page
- [ ] If BT Node, Additionally: add to BT's XML index of nodes for groot, BT package's readme table, and BT library lists

### CODE REVIEW

File: nav2_bringup/bringup/worlds/waffle.model [hunk 1]
```diff
@@ -137,6 +137,7 @@
             </linear_acceleration>
           </imu>
           <plugin name="turtlebot3_imu" filename="libgazebo_ros_imu_sensor.so">
+            <initial_orientation_as_reference>false</initial_orientation_as_reference>
```
Comment (SteveMacenski) [inline nav2_bringup/bringup/worlds/waffle.model:140]: This change is unrelated to the ticket, please revert

**Review by SteveMacenski — CHANGES_REQUESTED** [review]

File: nav2_bringup/bringup/worlds/waffle.model (hunk 1 above)
Comment (Darkproduct) [reply nav2_bringup/bringup/worlds/waffle.model]: I don't know how this line made its way into this. I certainly didn't add this intentionally. 

I remove it. This should now be ready for merge.

**Review by SteveMacenski — APPROVED** [review]
